src/containers/Form/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import ReceiverForm from "../../components/ReceiverForm";
+import DonatorForm from "../../components/DonatorForm";
+import { Container, ToggleButton, ButtonGroup, Row } from "react-bootstrap";
+import Logo from "../../logo.png";
+import "./index.css";
+
+export default function Form() {
+  const [radioValue, setRadioValue] = useState("1");
+
+  const radios = [
+    { name: "Receive", value: "1" },
+    { name: "Donate", value: "2" },
+  ];
+
+  return (
+    <Container>
+      <Row className="justify-content-md-center">
+        <img src={Logo} alt="fortune-found" className="formLogo" />
+      </Row>
+      <Row className="justify-content-md-center">
+        <ButtonGroup toggle>
+          {radios.map((radio, idx) => (
+            <ToggleButton
+              key={idx}
+              type="radio"
+              variant="secondary"
+              className="toggle"
+              name="radio"
+              value={radio.value}
+              checked={radioValue === radio.value}
+              onChange={(e) => setRadioValue(e.currentTarget.value)}
+            >
+              {radio.name}
+            </ToggleButton>
+          ))}
+        </ButtonGroup>
+      </Row>
+      {radioValue === "1" ? <ReceiverForm /> : <DonatorForm />}
```

Comment:
Here is where you can pass props to differentiate donator from receiver in the same form